src/components/Filter/Filter.tsx

```diff
@@ -123,36 +134,52 @@ export const Filter: FC<FilterComponentProps> = ({
                             ))}
                         <FilterIcon className="filterMenuIcon" onClick={toggleCollapsed} />
                     </FilterHeader>
+                    {standardMode && (
+                        <FiltersSection className='filters'>
+                            {Object.entries(filters)
+                                .filter(([, f]) => !f.inputHidden)
+                                .map(([key, filter]) => {
+                                    const itemFilterLabelValues = filters[key].getFilterSectionLabel(filterValues[key]);
 
-                    {(!isJSONInputAllowed || !jsonQueryParams.jsonView) && (
-                        <FiltersSection className="filters">
+                                    return (
+                                        <FilterValueChips
+                                            label={filters[key].label}
+                                            clearFilter={clearFilter}
+                                            key={key}
+                                            name={key}
+                                            notDefaultValues={filters[key] ? !filters[key].isDefaultFilterValue(filterValues[key]) : false}
+                                            value={itemFilterLabelValues}
+                                            visible={true} />
+                                    );
+                                })}
+                        </FiltersSection>
```

Comment:
There should be a border of some sort at the bottom of this (between the chips and the first filter section). See the original mockup to see what I mean

![image](https://user-images.githubusercontent.com/80778550/214612298-b8f4cc8a-cd4a-4a69-ac16-d1a887e880ef.png)